test/functional/tiertwo_dkg_pose.py

```diff
@@ -32,15 +32,18 @@ def run_test(self):
         assert_equal(len(self.mns), 6)
 
         # Mine a LLMQ final commitment regularly with 3 signers
-        _, bad_mnode = self.mine_quorum()
+        qfc, bad_mnode = self.mine_quorum()
         assert_equal(171, miner.getblockcount())
         assert bad_mnode is None
+        self.check_final_commitment(qfc, valid=[1, 1, 1], signers=[1, 1, 1])
 
         # Mine the next final commitment after disconnecting a member
-        _, bad_mnode = self.mine_quorum(invalidate_func=self.disable_network_for_node,
-                                        skip_bad_member_sync=True)
+        dkg_msgs = [ExpectedDKGMessages(True, True, False, True, 2, 2, 0, 2) for _ in range(2)]
```

Comment:
nit: would be good to use keyword arguments when we have long constructors like this one.